libs/platform/experience/src/models/composition.model.ts

```diff
@@ -208,6 +209,13 @@ export interface StyleProperties extends LayoutStylesProperties {
    */
   scale?: number;
 
+  /**
+   * Specifies the typography tag, which is one of the available heading tags.
+   * The typography is applied to the component and all it's sub components, including those components
+   * with a shadow dom, since typography is inherited in shadow dom.
+   */
+  typo?: HeadingTag;
```

Comment:
Suggest to use the full word here to exclude incorrect interpretations. Maybe it's even better to use rename it for better understanding - `defaultTypographyTag` for example.